sms/dlr-flask.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, request, jsonify
+from pprint import pprint
+app = Flask(__name__)
```

Comment:
A couple of blank lines between the imports and the app definition would probably be more PEP-8 compliant :)